MDM/AppConfig/README.md

```diff
@@ -0,0 +1,21 @@
+## AppConfig how to
+
+### Managed App Config
+Starting with iOS7, Apple has added support for managed application configuration. MDM server can push configuration to the app. The app can the access the configuration using NSUserDefaults class. The configuration is basically a key-value dictionary provided as .plist file.
```

Comment:
Just a typo:
Remove `the` before `access`

The app `can access` the configuration using NSUserDefaults class.

Reply:
@hosy Thanks for pointing out. Fixed.